### PR TITLE
Add an OWNERS file for gopherage

### PR DESCRIPTION
**Pull request by Katharine**
Add myself as an approver to gopherage. Also, auto-assign the label.

/area gopherage
/cc @BenTheElder

### CODE REVIEW

File: gopherage/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+approvers:
+- ktbry
```
Comment (BenTheElder) [inline gopherage/OWNERS]: lol whoops

Comment (krzyzacy) [reply gopherage/OWNERS]: /shrug

Comment (Katharine) [reply gopherage/OWNERS]: what even are usernames anyway

**Comment by krzyzacy** [comment]
oh `this-is-unbeaeable` I need auto-fix :-\

/this-is-unbearable

**Pull request merged**